extensions.json

```
@@ -93,6 +93,12 @@
       "repository": "https://github.com/appulate/vscode-file-watcher",
       "version": "1.0.3"
     },
+    {
+      "id": "arcanis.vscode-zipfs",
+      "repository": "https://github.com/yarnpkg/berry.git",
```

While this should work as is, I personally prefer repository URLs without the unnecessary `.git` suffix:

```suggestion
      "repository": "https://github.com/yarnpkg/berry",
```